.github/workflows/build-windows.yml

```diff
@@ -120,11 +120,15 @@ jobs:
           cmake --build build -j4 --config ${{ matrix.build_type }} --target check.inference
           cmake --build build -j4 --config ${{ matrix.build_type }} --target check.linear
           cmake --build build -j4 --config ${{ matrix.build_type }} --target check.navigation
+          cmake --build build -j4 --config ${{ matrix.build_type }} --target check.navigation2
```

Comment:
Hi @talregev can you please explain what all the `check.<group>2` tests are? I don't believe they are defined anywhere in the repository.

Reply:
I define in my fix tests that are conflicting with each other. I think there is a bug / feature in Microsfot c++ compiler, that it cannot have the same definition of template function/class in several mains.

Reply:
For example for navigation2:
https://github.com/borglab/gtsam/pull/1581/files#diff-f94c8bb2f8d261ad6f3be912ac13bab6f826a9071bdecab27ffabe8a2f1d2fc6R35

Reply:
https://stackoverflow.com/questions/44960760/msvc-dll-exporting-class-that-inherits-from-template-cause-lnk2005-already-defin

Reply:
revert.